Source/animation/types/ShapeAnimation.swift

```diff
@@ -7,7 +7,9 @@
 //
 
 class ShapeAnimation: AnimationImpl<Shape> {
-    convenience init(animatedNode: Shape, finalValue: Shape, animationDuration: Double, delay: Double = 0.0, autostart: Bool = false, fps: UInt = 30) {
+    let toParentGlobalTransfrom: Transform?
```

Comment:
Again no need to use optional property when you have great default value and it's actually used everywhere.